### PR TITLE
Gherkin scenarios for language switching

### PR DESCRIPTION
**Pull request by nucleogenesis**
### Summary
Adds gherkin scenarios for all user types changing their language prior to signing in.

### Reviewer guidance

[In reference to this Notion page](https://www.notion.so/learningequality/Personal-Dashboard-edc0356e45ee4700ae3b8f349537beea?p=f8c0837f9a254f24bf623a788897bd23) - ensure that all scenarios and stories are covered for all user types so that we are covering all expected user stories.

### References

[Notion Page](https://www.notion.so/learningequality/Personal-Dashboard-edc0356e45ee4700ae3b8f349537beea?p=f8c0837f9a254f24bf623a788897bd23)

### Contributor Checklist

PR process:

- [x] PR has the correct target branch and milestone
- [x] PR has 'needs review' or 'work-in-progress' label
- [x] If PR is ready for review, a reviewer has been added. (Don't use 'Assignees')
- [ ] If this is an important user-facing change, PR or related issue has a 'changelog' label
- [ ] If this includes an internal dependency change, a link to the diff is provided

Testing:

- [ ] Contributor has fully tested the PR manually
- [ ] If there are any front-end changes, before/after screenshots are included
- [ ] Critical user journeys are covered by Gherkin stories
- [ ] Critical and brittle code paths are covered by unit tests

### Reviewer Checklist

- Automated test coverage is satisfactory
- PR is fully functional
- PR has been tested for [accessibility regressions](http://kolibri-dev.readthedocs.io/en/develop/manual_testing.html#accessibility-a11y-testing)
- External dependency files were updated if necessary (`yarn` and `pip`)
- Documentation is updated
- Contributor is in AUTHORS.md

### CODE REVIEW

File: integration_testing/features/admin/admin-change-language.feature [hunk 1]
```diff
@@ -4,6 +4,20 @@ Feature: Admin changes user interface language
...
+    Then Kolibri is in <language>
+    When I log out
+    Then I am redirected to the sign in page and Kolibri is in <language>
+    When I open a new tab and open Kolibri
+    Then Kolibri is shown in <language>
+    When I refresh the page
+    Then Kolibri is shown in <language>
+    When I navigate between and within Coach, Learn, and Facility sections of Kolibri
+    Then Kolibri remains in <language>
+    When I close my browser altogether and go to Kolibri's root server URL
```
Comment (radinamatic) [inline integration_testing/features/admin/admin-change-language.feature]: @nucleogenesis In my VM at least, both FF & Chrome browsers retain the language used by the user who last visited Kolibri, even when it's different from the <device_language>. So atm, the only way to actually get Kolibri to display in the set <device_language>, is to open the incognito window. For clearer test case, I suggest you change the above `When` accordingly, as just closing, reopening the browser and going to the root URL won't do the trick.

Comment (radinamatic) [reply integration_testing/features/admin/admin-change-language.feature]: Do the above in all 4 scenarios.

**Review by radinamatic — APPROVED** [review]
If _defaulting back to the device language after the user closes the browser out_ is the intended behavior (it does not work atm on `0.12.6.b1` VM), then this is good to merge!

File: integration_testing/features/admin/admin-change-language.feature [hunk 2]
```diff
@@ -4,6 +4,20 @@ Feature: Admin changes user interface language
   Background:
     Given I am signed in to Kolibri as facility admin user
 
+  Scenario: Admin has changed language from <device_language> to <language> prior to logging in
```
Comment (indirectlylit) [inline integration_testing/features/admin/admin-change-language.feature]: might want to clarify in this and the other four stories

> Admin has changed **_their own_** language from

Comment (indirectlylit) [reply integration_testing/features/admin/admin-change-language.feature]: also, this seems like maybe it should go below the story below?

**Comment by radinamatic** [comment]
This is the recording of the current 0.12.6.b1 state on Ubuntu VM

default-language2

**Comment by radinamatic** [comment]
First window (Firefox), admin logged in, set the `default_language` to Spanish.

Second window (Chrome), learner logged in, set their own language as Arabic.

Third window (Chromium), no user logged in before, and the Kolibri opens in `default_language`.

As soon as the user sets their own language, the next tab or window opened with Kolibri IP in that browser defaults to the language used previously.

The only way to see the `default_language` is to open the browser incognito.

**Comment by radinamatic** [comment]
Browser or Kolibri retain the previous user language selection, so even if a **new user logs in for the first time**, they will have to change the language themselves, not even clearing all the cookies and browsing data work for Kolibri to display the `default_language`.

**Comment by indirectlylit** [comment]
follow-up in https://github.com/learningequality/kolibri/issues/5794

**Pull request merged**